### DISCUSSION TITLE
Can't get github student developer pack

### DISCUSSION THREAD

**Discussion by Shahar-Yar095** [community/community#57215] [GitHub Education]
The inability to obtain the GitHub Student Developer Pack has hindered my access to valuable resources and benefits offered by the program. Despite diligently following the eligibility criteria and application process, the lack of response from GitHub Support has left the issue unresolved. It is crucial for GitHub to address this problem promptly to ensure a positive user experience and enable students to leverage the benefits of the Student Developer Pack.

**Comment by samus-aran** [comment]
Hey @Shahar-Yar095  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!
